src/main/java/finalcountdown/homerecipesystembackend/controller/PrepMethodController.java

```diff
@@ -0,0 +1,73 @@
+package finalcountdown.homerecipesystembackend.controller;
+
+import finalcountdown.homerecipesystembackend.model.*;
+import finalcountdown.homerecipesystembackend.service.PrepMethodService;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@Slf4j
+@RequestMapping("/prep-method")
+public class PrepMethodController {
+    private final PrepMethodService prepMethodService;
+
+    public PrepMethodController(PrepMethodService prepMethodService) {
+        this.prepMethodService = prepMethodService;
+    }
+
+    @PostMapping("/create")
+    public ResponseEntity<?> createPrepMethod(@RequestBody PrepMethod newPrepMethod) {
+        log.info("New prepMethod to save", newPrepMethod);
+        prepMethodService.savePrepMethod(newPrepMethod);
+        return ResponseEntity.created(URI.create("/unit/create/%d"
+                        .formatted(newPrepMethod.getId())))
+                        .body(newPrepMethod);
+    }
+
+    @PostMapping("/add-step-to-prep-method")
+    public ResponseEntity<Void> addStepToPrepMethod(Step step) {
```

Comment:
We need to test this out if we can use Step object here or we still need to create a StepRequest for here too..